docs/technical-details/agent/collecting-diagnostics.md

```diff
@@ -0,0 +1,11 @@
+# Collecting Diagnostics
```

Comment:
I think we'll need to add this to the routes here: https://github.com/FairwindsOps/insights-docs/blob/ba5cd05f385af1dfb7f42d31074c38d39201fc79/docs/.vuepress/config-extras.js#L48